web_responsive/static/src/css/web_responsive.scss

```diff
@@ -8,9 +8,10 @@ $chatter_zone_width: 35%;
     box-shadow: none;
     display: flex;
     flex-direction: column;
-    height: calc(100vh - #{$o-navbar-height});
-    max-height: calc(100vh - #{$o-navbar-height});
+    height: calc(100vh - #{$o-navbar-height}) !important;
+    max-height: calc(100vh - #{$o-navbar-height}) !important;
```

Comment:
Usually there will be theming problems in submodules when we use `!important`. Is it possible to avoid that here? Maybe by using a more specific CSS selector...

If not possible, always put a comment explaining why `!important` is required.